lucien-ios/Models/DashboardComic.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  DashboardComic.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 10/18/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import Foundation
+
+enum DashboardComicError: Error {
+    case noUser
+    case noComics
+}
+
+struct DashboardComic: Decodable {
+    var dashboardComic: Comic
+    var dashboardUser: User
+    var dashboardUserType: String
+
+    enum CodingKeys: String, CodingKey {
+        case borrower
+        case owner
+    }
+
+    init(dashboardComic: Comic, dashboardUser: User, dashboardUserType: String) {
+        self.dashboardComic = dashboardComic
+        self.dashboardUser = dashboardUser
+        self.dashboardUserType = dashboardUserType
+    }
+
+    init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: Comic.CodingKeys.self)
+        let id = try container.decode(String.self, forKey: .comicID)
+        let createdAt = try? container.decode(Date.self, forKey: .createdAt)
+        let updatedAt = try? container.decode(Date.self, forKey: .updatedAt)
+        let ownerID = try container.decode(String.self, forKey: .ownerID)
+        let borrowerID = try? container.decode(String.self, forKey: .borrowerID)
+        let comicTitle = try container.decode(String.self, forKey: .comicTitle)
+        let storyTitle = try? container.decode(String.self, forKey: .storyTitle)
+        let volume = try? container.decode(String.self, forKey: .volume)
+        let issueNumber = try? container.decode(String.self, forKey: .issueNumber)
+        let publisher = try? container.decode(String.self, forKey: .publisher)
+        let releaseDate = try? container.decode(Date.self, forKey: .releaseDate)
+        let comicPhotoURL = try? container.decode(String.self, forKey: .comicPhotoURL) 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Trailing Whitespace Violation: Lines should not have trailing whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atrailing_whitespace)